base/boot.jl

```diff
@@ -260,8 +260,12 @@ struct AssertionError <: Exception
 end
 AssertionError() = AssertionError("")
 
-#Generic wrapping of arbitrary exceptions
-#Subtypes should put the exception in an 'error' field
+"""
+    WrappedException([msg])
+
+wrap arbitrary exceptions generically.
```

Comment:
Generic type for `Exception`s wrapping another `Exception`, such as `LoadError` and `InitError`.
Those exceptions contain information about the the root cause of an exception. Subtypes define a field `error` containing the causing`Exception`.